apps/wellknown/views.py

```diff
@@ -59,9 +66,11 @@ def build_endpoint_info(data=OrderedDict(), issuer=""):
     """
     data["issuer"] = issuer
     data["authorization_endpoint"] = issuer + \
-        reverse('oauth2_provider:authorize')
-    data["token_endpoint"] = issuer + reverse('oauth2_provider:token')
-    data["userinfo_endpoint"] = issuer + reverse('openid_connect_userinfo')
+        reverse_sin_trailing_slash('oauth2_provider:authorize')
```

Comment:
I'm concerned about patching this issue up front instead of fixing the root problem. Why are the trailing slashes there in the first place?